### PR TITLE
fix(core): vue props spread

### PR DESCRIPTION
**Pull request by kingzez**
## Description

Add a short description of:
- what changes you made https://github.com/BuilderIO/mitosis/issues/869

### CODE REVIEW

**Comment by kingzez** [comment]
Need I update snapshot?

**Comment by samijaber** [comment]
Yup, most likely. `yarn snapupdate` in `packages/core`

File: packages/core/src/generators/vue/blocks.ts [hunk 1]
```diff
@@ -376,7 +376,7 @@ export const blockToVue: BlockRenderer = (node, options, scope) => {
       let spreadsString = `{...${spreads.join(', ...')}}`;
       str += ` v-bind="${encodeQuotes(spreadsString)}"`;
     } else {
-      str += ` v-bind="${encodeQuotes(spreads.join(''))}"`;
+      str += ` v-bind="$${encodeQuotes(spreads.join(''))}"`;
```
[the remark this answers is not in the document]
Comment (kingzez) [reply packages/core/src/generators/vue/blocks.ts]: Fixed

**Comment by kingzez** [comment]
1. fixed: only props spread add $
2. update snapshot

**Review by samijaber — APPROVED** [review]
Excellent, thank you!

**Pull request merged**